templates/Generalized_Masking.ipynb

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
<p>Personally I prefer leaving <code>sigma</code>, <code>min_mask_size</code>, and <code>max_hole_size</code> as hidden default arguments inside <code>generate_signal_masks</code>. If you expect the user will change these, then I'd specify them as customizable variables in the cell above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>The defaults set here are currently based only on what Noah used, so I don't want users to think the masks generated with them are the only output possible. I'll add them to the description and set them in the cell above.</p>